docusaurus/docs/ui-extensions/create-an-extension-point.md

```diff
@@ -15,40 +15,288 @@ keywords:
   - apps
 ---
 
-Providing an extension point within your UI enables other plugins to contribute supplementary capabilities, and leverage the context within the current view to guide users to a potential next step. Here's what you need to know about extension points.
 
-Firstly, you will need to define an extension point ID. This is basically just a string describing the part of the UI where the extension point lives. Extension developers should be able to figure out where in the UI the extension will be added by reading the extension point ID.
+In this guide you will learn how to provide an extension point so that other app plugins can add their extensions to your plugin.
 
-:::note
+## What is an extension point?
 
-Extension points living in core Grafana must start with `grafana/` and extension points living in plugins must have IDs starting with `plugins/` followed by the plugin ID, for example, `plugins/<PLUGIN_ID>/`.
+An extension point is a place in the UI which you make extendable by other plugins using extensions. These extensions can be either be links or simple React components (that can basically impelement anything). It's up to you how you are displaying the handling or displaying the extensions, we have some examples below.
```

Comment:
I'm not 100% sure we need to define it _again_, but not a blocking comment https://grafana.com/developers/plugin-tools/ui-extensions/#definition-of-terms

Building on @josmperez 's suggestion above and incorporating your response @leventebalogh 
```suggestion
An extension point is a place in the UI which you make extendable by other plugins using extensions. These extensions can either be links or React components that can implement virtually anything. 

In the extension point you can control how you want to handle the displaying of extensions to users. You can also share contextual information with the extensions using a `context` object or component props. Refer to the examples below to learn more.
```
An Examples subsection might be useful to break up the content and provide us a way to link from here. Note - I've also moved the content from below as we had two sections both ending with "see the examples"